readthedocs/vcs_support/backends/git.py

```diff
@@ -199,12 +199,16 @@ def fetch(self):
             "--depth",
             str(self.repo_depth),
         ]
-        remote_reference = self.get_remote_fetch_refspec()
-
-        if remote_reference:
-            # TODO: We are still fetching the latest 50 commits.
-            # A PR might have another commit added after the build has started...
-            cmd.append(remote_reference)
+        is_rtd_latest = (
+            self.verbose_name == LATEST_VERBOSE_NAME and self.version_machine
+        )
```

Comment:
This logic is repeated 5 times in this PR... should it just be a method on `self` here, so that we don't have it defined in different ways?